01-js-intro-and-jquery/07/js/scripts.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+	var alias = '';
```

Comment:
This variable should be defined within ``getResponse`` method, since that would be its correct scope (right now is living in your ``ready`` method and you want your variable to be accessible just by ``getResponse`` method).